Article/Article.js

```diff
@@ -70,7 +70,7 @@ function buildArticle(headlineIn, dateIn, ...text) {
     div.appendChild(paragraph);
   });
   const expand = document.createElement('span');
-  expand.classList.add('expandButton');
+  expand.classList.add('expand-button');
   div.appendChild(expand);
   document.querySelector('.articles').appendChild(div);
```

Comment:
Fantastic job getting this stretch going! adding the ability to add new content via a form is such a big thing!